cogboard-webapp/src/components/UserLogin.js

```diff
@@ -46,6 +47,20 @@ const UserLogin = () => {
     openLogoutSnackbar();
   };
 
+  const getCredentials = () => {
```

Comment:
Move it to helpers. Component should have render and event handlers, nothing more.

Reply:
Done :)